packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
My concern with keeping it in our repo is that we need to enforce the props-table to be updated when we change src code. We would need to add a prepush script to keep the src files in sync with the props-table.

I was going to make a consistent naming scheme except we have some components that have multiple props tables ([list](https://github.com/cerner/terra-core/tree/props-table-cli/packages/terra-list/docs) for example).